src/scss/06_components/navigation/_branding-navigation.scss

```diff
@@ -21,18 +21,18 @@ $-branding-icon-size-desktop: $spacing-double;
     --branding-height: 86px;
     --header-branding-padding: #{$spacing-and-half};
     grid-template-areas:
-      '. logo . lang brand search toggle .'
-      'langmenu langmenu langmenu langmenu langmenu langmenu langmenu langmenu'
-      'menu menu menu menu menu menu menu menu';
-    grid-template-columns: var(--header-branding-padding) auto 1fr auto auto auto auto var(--header-branding-padding);
+      '. logo . lang brand search toggle profile .'
+      'langmenu langmenu langmenu langmenu langmenu langmenu langmenu langmenu langmenu'
+      'menu menu menu menu menu menu menu menu menu';
+    grid-template-columns: var(--header-branding-padding) auto 1fr auto auto auto auto auto var(--header-branding-padding);
     grid-template-rows: var(--branding-height) auto;
   }
 
   @include breakpoint($breakpoint-l) {
     grid-template-areas:
-      '. logo . lang brand search toggle .'
-      'langmenu langmenu langmenu langmenu . . . .'
-      'menu menu menu menu menu menu menu menu';
+      '. logo . lang brand search toggle profile .'
+      'langmenu langmenu langmenu langmenu langmenu . . . .'
```

Comment:
Here is a bug. The dots on the right here should match the grid element count after `lang` element. The current approach will cause an alignment problem when(/if) brand-menu is re-introduced. We want to set the `langmenu` to match its right corner with its `lang` button.
![image](https://user-images.githubusercontent.com/1191667/214817392-f9033d11-0391-49cc-bcfd-dbb7eeffc501.png)

So, on this row there should be 5 dots. Change the last `langmenu` to a dot.

Reply:
Nice one! Fixed this.